docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -540,7 +540,7 @@ in the metastore. As a result, any Databricks engine can write to the table::
 The Delta Lake connector also supports creating tables using the :doc:`CREATE
 TABLE AS </sql/create-table-as>` syntax.
 
-There are three table properties available for use in table creation.
+There are four table properties available for use in table creation.
```

Comment:
Do we leave the necessary changes for updating this property for a follow-up PR?

Reply:
What necessary changes?

Reply:
See https://trino.io/docs/current/connector/iceberg.html#alter-table-set-properties for Iceberg equivalent.

The user should be able to eventually enable  and disable the CDF for a table.

Reply:
I would recommend adding such support in follow-up PR after #15932